src/components/evm/inputs/CurrencyInput.vue

```diff
@@ -1097,10 +1099,10 @@ export default defineComponent({
 
         position: absolute;
         top: -24px;
-        left: 0;
         right: 0;
-        text-align: right;
+        width: max-content;
         cursor: pointer;
+        padding: 0 0 8px 8px; // make clickable area slightly larger
```

Comment:
fixing an issue where the max amount container is clickable all the way to the left of the input. now only the text is clickable